proposals/lip-0052.md

```diff
@@ -1322,6 +1346,83 @@ def transferInternal(
     )
 ```
 
+#### verifyTransferCrossChainInternal
+
+```python
+def verifyTransferCrossChainInternal(
+    senderAddress: Address,
+    recipientAddress: Address,
+    nftID: NFTID,
+    receivingChainID: ChainID,
+    messageFee: uint64,
+    data: str,
+    includeAttributes: bool,
+    emitEvent: bool
+) -> None:
+
+    nft = getNFT(nftID)
```

Comment:
I think we should handle the possible exception from this function by emitting an event with `RESULT_NFT_DOES_NOT_EXIST` if the call to `getNFT` fails.

Reply:
Done in [this commit](https://github.com/LiskHQ/lips/pull/477/commits/876ae2c110612053e85a0c54fb65574a7e1b4bc1).